src/js/index.js

```diff
@@ -1,7 +1,74 @@
+const baseUrl = "https://js-todo-list-9ca3a.df.r.appspot.com";
+
+
 const onUserCreateHandler = () => {
-  const userName = prompt("추가하고 싶은 이름을 입력해주세요.");
+    const userName = prompt("추가하고 싶은 이름을 입력해주세요.");
+    if (userName == null || userName == undefined)
+        return;
+    if (userName.length < 2)
+        alert("사용자 이름은 최소 2글자 이상입니다.");
```

Comment:
```suggestion
    if (userName == null || userName == undefined)  return;
    (userName.length < 2) && alert("사용자 이름은 최소 2글자 이상입니다.");
```
if 조건이 처리하는 연산이 하나면, 한줄로 표현이 가능합니다. 
논리 연산자(&&)를 사용하셔도 가능해요!
저는 깔끔해서 선호하는 편이랍니다 😁